src/components/CreateExpenseModal/CreateExpenseModal.jsx

```diff
@@ -5,16 +5,41 @@ import Button from "../Button";
 import "./CreateExpenseModal.css";
 
 const CreateExpenseModal = (props) => {
-    return props.isShown === true ? (
-        <Modal className="expense-modal" onCloseClick={props.onCloseClick}>
-            <Title>Add new expense</Title>
-            <div className="expense-modal__form">
-                <Input className="expense-modal__input expense-modal__input--name" placeholder="Expense name"/>
-                <Input className="expense-modal__input expense-modal__input--amount" placeholder="$ Amount" type="number"/>
-                <Button variant="default" icon="plus"/>
-            </div>
-        </Modal>
-    ) : null;
-}
+  const handleExpenseSubmit = (event) => {
+    event.preventDefault();
+    const titleValue = event.target["title"].value;
+    const amountValue = event.target["amount"].value;
```

Comment:
There is some code duplication in here. `event.target["title"].value` and `<Input name="title"` could be changed to:

```
  const TITLE_NAME = "title";
...
  const CreateExpenseModal = ...
...
    const handleExpenseSubmit = ...
      const titleValue = event.target[TITLE_NAME].value
      ....
...
    return ( 
       ...
       <Input name={TITLE_NAME}...
... 
```

That way you can change "title" for any other string in `TITLE_NAME` and nothing would break because "title" is only defined in one place (centralised)

Reply:
I would do the same with amount. Replace "amount" with `AMOUNT_NAME` as specified above